tee-worker/omni-executor/aa-contracts/src/accounts/OmniAccountV1.sol

```diff
@@ -92,11 +108,11 @@ contract OmniAccountV1 is BaseAccount, TokenCallbackHandler, UUPSUpgradeable, In
         emit AccountInitialized(_entryPoint, owner, ownerType, clientId, aRoot);
     }
 
-    // Require the function call went through EntryPoint or be signed by [owner|root]
+    // Require the function call went through EntryPoint or be signed by owner
     function _requireForExecute() internal view virtual override {
         require(
-            msg.sender == address(entryPoint()) || _determineOa(msg.sender) == owner || isRootSigner(msg.sender),
-            "account: not Owner or EntryPoint or root"
+            msg.sender == address(entryPoint()) || msg.sender == address(this) || _determineOa(msg.sender) == owner,
```

Comment:
When would `msg.sender == address(this)` be checked?

Shall we remove it for simplicity?